src/app/screens/Receive/index.tsx

```diff
@@ -174,7 +174,7 @@ function Receive() {
                     navigator.clipboard.writeText(invoice.paymentRequest);
                     setCopyLabel(tCommon("copied"));
                     setTimeout(() => {
-                      setCopyLabel(tCommon("actions.copy"));
+                      setCopyLabel(tCommon("actions.copy")+" Invoice");
```

Comment:
That doesn't work this way. You'd need to create a new [translation](https://github.com/getAlby/lightning-browser-extension/blob/master/src/i18n/locales/en/translation.json#L1) for that instead of concatenating a string here.

Reply:
Thank you for guiding me, I will try to do by the way you suggested